### PR TITLE
Update stream test for cpe

### PR DESCRIPTION
**Pull request by ekouts**
https://jira.cscs.ch/browse/VCUE-35

### CODE REVIEW

File: checks/microbenchmarks/cpu/stream/stream.py [hunk 1]
```diff
@@ -14,94 +14,64 @@ class StreamTest(rfm.RegressionTest):
...
 
-    def __init__(self):
-        self.descr = 'STREAM Benchmark'
-        self.exclusive_access = True
-        self.valid_systems = ['daint:gpu', 'daint:mc', 'dom:gpu', 'dom:mc',
-                              'arolla:cn', 'arolla:pn', 'tsa:cn', 'tsa:pn']
-        self.valid_prog_environs = ['PrgEnv-cray', 'PrgEnv-gnu',
-                                    'PrgEnv-intel', 'PrgEnv-pgi',
-                                    'PrgEnv-nvidia']
+    descr = 'STREAM Benchmark'
```
Comment (jgphpc) [inline checks/microbenchmarks/cpu/stream/stream.py:17]: I would:
1/ update the Copyright 2016-2022 to 2016-2023
2/ add a reference to https://www.cs.virginia.edu/stream/FTP/Code/ (after line 14) to make it clear where the code is from

File: config/manali/cpe.py [hunk 2]
```diff
@@ -161,7 +161,7 @@
             'extras': {
                 # PrgEnv-nvidia requires --mpi=pmi2 at runtime:
                 # "MPIR_pmi_init(83)....: PMI2_Job_GetId returned 14"
-                'launcher_options': '--mpi=pmi2',
+                'launcher_options': ['--mpi=pmi2'],
```
[the remark this answers is not in the document]
Comment (ekouts) [reply config/manali/cpe.py:164]: In systems we keep the configs that were included in the old `cscs.py`. Now they are automatically imported. @teojgo/@jgphpc do we know where/if this configuration is used in any pipeline?

File: checks/microbenchmarks/cpu/stream/stream.py [hunk 3]
```diff
@@ -14,94 +14,64 @@ class StreamTest(rfm.RegressionTest):
...
+            'hohgant:nvgpu': {'triad': (511500, -0.05, None, 'MB/s')},
+            'hohgant:cpu': {'triad': (1791161, -0.05, None, 'MB/s')}
+        },
+        'PrgEnv-nvidia': {
+            'hohgant:nvgpu': {'triad': (477077, -0.05, None, 'MB/s')},
+            'hohgant:cpu': {'triad': (1804001, -0.05, None, 'MB/s')}
+        }
+    }
 
-        self.use_multithreading = False
```
Comment (jgphpc) [inline checks/microbenchmarks/cpu/stream/stream.py:52]:
```suggestion
```

File: checks/microbenchmarks/cpu/stream/stream.py [hunk 4]
```diff
@@ -14,94 +14,64 @@ class StreamTest(rfm.RegressionTest):
...
-            self.current_partition.fullname, 1)
+        self.skip_if_no_procinfo()
+        self.num_cpus_per_task = int(self.current_partition.processor.num_cores)
         self.env_vars['OMP_NUM_THREADS'] = self.num_cpus_per_task
-        envname = self.current_environ.name
 
-        self.build_system.cflags = self.prgenv_flags.get(envname, ['-O3'])
-        if envname == 'PrgEnv-pgi':
-            self.env_vars['OMP_PROC_BIND'] = 'true'
+        self.build_system.cflags += self.current_environ.extras.get('openmp_flags', [])
```
Comment (jgphpc) [inline checks/microbenchmarks/cpu/stream/stream.py:74]: line too long

**Review by jgphpc — APPROVED** [review]

File: config/manali/cpe.py (hunk 2 above)
[the remark this answers is not in the document]
Comment (teojgo) [reply config/manali/cpe.py:164]: Yeah these are the old ones. We should remove at some point

[the remark this answers is not in the document]
Comment (ekouts) [reply config/manali/cpe.py:164]: Ok thanks. Let's go through the configuration files as well on the merging meeting and get rid of any leftovers that are not used.

**Pull request merged**